src/components/NotificationsDrawer/NotificationItem.tsx

```diff
@@ -83,7 +85,9 @@ const NotificationItem: React.FC<NotificationItemProps> = ({ notification, onNav
             <Label variant="outline" isCompact className="pf-u-mb-md">
               {notification.source}
             </Label>
-            <span className="pf-u-display-block">{notification.description}</span>
+            <span className="pf-u-display-block">
+              <ReactMarkdown remarkPlugins={[remarkGfm]}>{notification.description}</ReactMarkdown>
```

Comment:
Is the content always a markdown syntax?

Reply:
Yes, notifications team is sending markdown for all notifications.